scripts/update/lidarr.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+if [[ -f /install/.lidarr.lock ]]; then
+
+    if [ -z "$LIDARR_OWNER" ]; then
+        if ! LIDARR_OWNER="$(swizdb get lidarr/owner)"; then
+            LIDARR_OWNER=$(_get_master_username)
+            swizdb set "lidarr/owner" "$LIDARR_OWNER"
+        fi
+    else
+        swizdb set "lidarr/owner" "$LIDARR_OWNER"
+    fi
+
+    #Move old homedirectory installations to opt and switch to netcore
+    user="$LIDARR_OWNER"
+    if [[ -d /home/$user/Lidarr ]]; then
+        echo_info "Moving Lidarr instllation to opt and switching it to net-core"
+        wasActive=$(systemctl is-active lidarr)
+        systemctl stop lidarr
+
+        echo_progress_start "Moving ~/Lidarr to /root/swizzin/backups/lidarr-mono"
+        mkdir -p /root/swizzin/backups
+        mv /home/"$user"/Lidarr /root/swizzin/backups/lidarr-mono || {
+            echo_error "Move failed, please investigate. exiting."
+            exit 1
+        }
+        echo_progress_done "Directory moved"
+
+        echo_progress_start "Downloading netcore binaries"
+        urlbase="https://lidarr.servarr.com/v1/update/develop/updatefile?os=linux&runtime=netcore"
+        case "$(_os_arch)" in
+            "amd64") dlurl="${urlbase}&arch=x64" ;;
+            "armhf") dlurl="${urlbase}&arch=arm" ;;
+            "arm64") dlurl="${urlbase}&arch=arm64" ;;
+            *)
+                echo_error "Arch not supported"
+                exit 1
+                ;;
+        esac
+
+        if ! curl "$dlurl" -L -o /tmp/lidarr.tar.gz >> "$log" 2>&1; then
+            echo_error "Download failed, exiting"
+            exit 1
+        fi
+        echo_progress_done "Binaries downloaded"
+
+        echo_progress_start "Extracting source"
```

Comment:
we're not using source;  binaries or release

Reply:
```suggestion
        echo_progress_start "Extracting archive"
```